packages/paste-website/src/pages/content/product-style-guide.mdx

```diff
@@ -0,0 +1,312 @@
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings}/>
+
+<content>
+
+## Content principles
+
+#### Identify your user and speak directly to them
+- Consider who you're writing for and their technical level – use words they're familiar with. 
+- Address the user as you, refer to Twilio as we.
+
+#### Write for the reading level of a 10-13 year old
+- Use simple, short sentences and common words. 
+- Explain technical terms and write acronyms in full the first time a user sees them.  
+
+<Callout>
+  <CalloutTitle>Using readability tools</CalloutTitle>
+  <CalloutText>
+    Sites offering <a href="https://readabilityformulas.com/free-readability-formula-tests.php">free readability tools like this one</a> can be useful for assessing the "reading age" of long content (more than 150 words). 
+    However, they're not great at testing short UI content that might only contain two or three words. 
+    Use them with caution. Look for a simpler word whenever you can to improve the readability of your content.   
+  </CalloutText>
+</Callout>
+
+#### Be inclusive of people and devices
+- Avoid *click, tap, touch* instructions as not every device supports touch. 
+- Instead, use *select* to instruct a user to interact.
+- Avoid relying on drectional or color-based cues. *To the right, the blue button* – are problematic for responsive products or users who see colors differently. 
+- Instead, write *the main menu* or *the dropdown menu*. Name the component or area you’re referencing.
+
+#### Write as if the user is in a rush
+- Avoid repetition and padding. 
+- Phrases we use when writing formally or presenting – *please note, so, in fact, in other words* – should be cut.
+- Chunk content and use headings to help the user scan for what they need.
+
+#### Be direct and straightforward
+- Lead with the reason (why), by front-loading messages with words that address the user’s needs.
+- Leave the user with a clear instruction of how to address that need. 
+- Avoid relying on company/industry jargon, idioms, or cultural references to communicate what you mean. 
+- Use the words that are most familiar to the user.
+
+#### It’s ok to bend grammatical rules to sound natural
+- End sentences with prepositions – *of, with, for* – if it sounds less awkward.
+- And you can start sentences with *But* or *And*. 
+- Use contractions with pronouns – *it’s, there’s, you’ll, we’ll, you’re* – to sound conversational. 
+
+<Callout>
+  <CalloutTitle>Exception: avoid contractions in error messages</CalloutTitle>
+  <CalloutText>
+    Avoid contractions in negative scenarios – when a user is stressed, their comprehension suffers. 
+    Often users will read *can’t* as *can*, missing the -’t at the end. 
+    Write out error messages or destructive action warnings in full.
```

Comment:
Same comment here about using "`*`" as in my other comment: These will have to use "`<em>...</em>`" or "`<strong>...</strong>`"

Reply:
Great - understood now, thanks.